datadog/dogstatsd/route.py

```diff
@@ -0,0 +1,32 @@
+"""
+Helper(s), resolve the system's default interface.
+"""
+# stdlib
+import socket
+import struct
+
+
+def get_default_route():
+    """
+    Return the system default interface using the proc filesystem.
+
+    Returns:
+        string: default route
+
+    Raises:
+        `NotImplementedError`: No proc filesystem is found (non-Linux systems)
+        `StopIteration`: No default route found
```

Comment:
Not sure to see the relation between "no default route found" and https://docs.python.org/2/library/exceptions.html#exceptions.StopIteration ? It doesn't seem to be used as an iterator anywhere, am I missing something ?